File: src/test/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/SpringApplicationContextLoadTest.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api;

import common.SpringIntegrationTestbase;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.sykefraværshistorikk.SykefraværshistorikkController;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;

import static org.assertj.core.api.Assertions.assertThat;

public class SpringApplicationContextLoadTest extends SpringIntegrationTestbase {

    @Autowired
    private SykefraværshistorikkController controller;

    @Test
    public void contexLoads() {
        assertThat(controller).isNotNull();
    }

}

File: src/test/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/SpringApplicationContextLoadTest.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api

import common.SpringIntegrationTestbase
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.api.SykefraværshistorikkController
import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired

class SpringApplicationContextLoadTest : SpringIntegrationTestbase() {
    @Autowired
    private val controller: SykefraværshistorikkController? = null
    @Test
    fun contexLoads() {
        Assertions.assertThat(controller).isNotNull()
    }
}
